lib/timers.js

```diff
@@ -94,8 +94,8 @@ const TIMEOUT_MAX = 2147483647; // 2^31-1
 //
 // - key = time in milliseconds
 // - value = linked list
-const refedLists = {};
-const unrefedLists = {};
+const refedLists = Object.create(null);
+const unrefedLists = Object.create(null);
```

Comment:
As was previously mentioned, I didn't measure the change specifically, but it does remove unnecessary hash table entries. It may also help if V8 sees only numeric keys being set on the object? It has nothing to do with key name collisions in this case.